bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -10306,7 +10374,8 @@ long getId() {
 		INSERT_VS_PART_SQLITE("INSERT INTO tsk_vs_parts (obj_id, addr, start, length, desc, flags) VALUES (?, ?, ?, ?, ?, ?)"),
 		INSERT_VS_PART_POSTGRESQL("INSERT INTO tsk_vs_parts (obj_id, addr, start, length, descr, flags) VALUES (?, ?, ?, ?, ?, ?)"),
 		INSERT_FS_INFO("INSERT INTO tsk_fs_info (obj_id, img_offset, fs_type, block_size, block_count, root_inum, first_inum, last_inum, display_name)"
-					+ "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)");
+					+ "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)"),
+		SELECT_CREATE_SCHEMA_VERSION("SELECT ");
```

Comment:
This SELECT_CREATE_SCHEMA_VERSION SQL statement needs to be either completed and used or deleted.

Reply:
removed